modules/oadp-1-4-0-release-notes.adoc

```diff
@@ -0,0 +1,64 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-06-28
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-0-release-notes_{context}"]
+= OADP 1.4.0 release notes
+
+The {oadp-first} 1.4.0 release notes lists resolved issues and known issues.
+
+[id="resolved-issues-1-4-0_{context}"]
+== Resolved issues
+
+.Restore works correctly in OpenShift Container Platform 4.16
+
+Previously, while restoring the deleted application namespace, the restore operation partially failed with the `resource name may not be empty` error in OpenShift Container Platform 4.16.
+With this update, restore works as expected in OpenShift Container Platform 4.16
+link:https://issues.redhat.com/browse/OADP-4075[OADP-4075]
+
+.Data Mover backups works properly in the OpenShift Container Platform 4.16 cluster
+
+Previously, Velero was using the lower version of SDK where the `Spec.SourceVolumeMode` field did not exist. As a consequence, Data Mover backups failed in the OpenShift Container Platform 4.16 cluster on the external snapshotter with v4.2 version. 
+With this update, external snapshotter is upgraded to v7.0 version and later. As a result, backups do not fail in the OpenShift Container Platform 4.16 cluster.
+link:https://issues.redhat.com/browse/OADP-3922[OADP-3922]
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
+
+.Backup fails when checksumAlgorithm is not set for MCG
+
+While performing a backup of any application with Noobaa as the backup location, if the `checksumAlgorithm` configuration parameter is not set, backup fails. To work around this problem, an empty value of `checksumAlgorithm` is added to the Backup Storage Location (BSL) configuration if you have not provided this value.
+The empty value is only added for BSLs that are created using Data Protection Application (DPA) custom resource (CR), and this value is not added if BSLs are created using any other method. 
+link:https://issues.redhat.com/browse/OADP-4274[OADP-4274]
+
+[id="upgrade-notes-1-4-0_{context}"]
+== Upgrade notes
+
+[NOTE]
+====
+Always upgrade to the next minor version. *Do not* skip versions. To update to a later version, upgrade only one channel at a time. For example, to upgrade from {oadp-first} 1.1 to 1.3, upgrade first to 1.2, and then to 1.3.
+====
+
+[id="changes-oadp-1-3-to-1-4_{context}"]
+=== Changes from OADP 1.3 to 1.4
+
+The Velero server has been updated from version 1.12 to 1.14.
+
+This changes the following:
+
+* The `velero-plugin-for-csi` code is now available in the Velero code, which means, no `init` container is needed for the plugin anymore. There are changes needed in the Data Protection Application (DPA).
```

Comment:
... There are **no** changes needed in the ... 

Missing the NO

Reply:
we are talking about the changes yes, but that phrase highlights that user does NOT need to change its DPA because of them

Reply:
done

Reply:
this is still unresolved

what we need is a NO in this phrase 

as I said, there are changes, but here we are highlighting to the user that even with the CSI plugin changes, DPA was not changed, which means, user does not need to change its DPA to use oadp 1.4 (as we also say in converting DPA section)

the last sentence in this line needs to be changed to `There are no changes needed in the Data Protection Application (DPA).`